README-ocp4.md

```diff
@@ -34,25 +33,25 @@ The Federation Operator requires at least one [OpenShift Container Platform](htt
 
 This walkthrough will use 2 OCP 4.0 clusters deployed using the [developer preview on AWS](https://cloud.openshift.com/clusters/install).
```

Comment:
Possibly change this to 4.1 as that will be the GA release version and its the current version in BETA